### ISSUE TITLE
release notes for v0.17.0 report wrong config version

### ISSUE THREAD

**Issue by davidstanke** [GoogleContainerTools/skaffold#1235]
The [release notes for version 0.17](https://github.com/GoogleContainerTools/skaffold/releases/tag/v0.17.0) say: "This release comes with a config change, use `skaffold fix` to upgrade your config to `v1alpha4`."

...actually, the output of v 0.17.0's `skaffold fix` renders version `v1alpha5`

**Comment by r2d4** [comment]
ref #1239. It's not clear which config versions correspond to which skaffold versions

**Comment by balopat** [comment]
Hmmm...I must have messed up the release of 0.17.0. 
Let me double check the mess I created.

**Comment by balopat** [comment]
OK. so 0.17.0 should have worked with v1alpha4 as the state was `latest=v1alpha4` on HEAD before the release. That's why we have currently v1alpha4 in the examples folder and the release notes is consistent with this intent as well. 

However, in a moment of confusion I merged David's PR at release day #1146 before the release - preparing for the next schema version with `latest=v1alpha5` - which should have been merged _after_ the release, and _only_ right before there is a schema impacting change, which could be actually 5 releases down the line if there is no schema change!!! :/

So basically, what we have is: 

- 0.17.0 was released with `latest=v1alpha5` but everything else (examples in master, release notes), is inconsistently pointing to v1alpha4 
- v1alpha4 === v1alpha5  (noop upgrade, except the version number)

Question is: do we have schema changes since the last 11 days? 

- if yes, then we need to prepare v1alpha6  (:()
- if not, we just need to copy the v1alpha5 integration/examples to examples and change the release notes to v1alpha5
